### PR TITLE
Comment out the sample to avoid error when not needed

### PR DESCRIPTION
**Pull request by coulof**
# Description
Commenting out the 2nd array per requirement here : https://github.com/dell/csm/issues/1030

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| 1030 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Not tested

### CODE REVIEW

File: samples/secret.yaml [hunk 1]
```diff
@@ -31,9 +31,11 @@
...
-- username: "admin"
-  password: "Password123"
-  systemID: "2b11bb111111bb1b"
-  endpoint: "https://127.0.0.2"
-  skipCertificateValidation: true 
-  mdm: "10.0.0.3,10.0.0.4"
+
+# # To add more PowerFlex systems, uncomment the following lines and provide the required values
+# - username: "admin"
+#   password: "Password123"
```
Comment (AronAtDell) [inline samples/secret.yaml:37]: Change "Password123" to "password" if we are just using an example.

**Review by AronAtDell — DISMISSED** [review]
Apart from my comment. Looks good. Thanks!

**Review by adarsh-dell — DISMISSED** [review]

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

File: samples/secret.yaml (hunk 1 above)
Comment (coulof) [reply samples/secret.yaml]: done

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

**Review by adarsh-dell — APPROVED** [review]

**Pull request merged**